mappings/net/minecraft/nbt/NbtList.mapping

```diff
@@ -1,4 +1,11 @@
-CLASS net/minecraft/class_2499 net/minecraft/nbt/ListTag
+CLASS net/minecraft/class_2499 net/minecraft/nbt/NbtList
+	COMMENT Represents an NBT list.
+	COMMENT <p>
+	COMMENT An NBT list holds values of the same type.
+	COMMENT An empty NBT list doesn't have an associated type,
+	COMMENT the type will be determined when the first element is inserted.
```

Comment:
```suggestion
	COMMENT The {@linkplain AbstractNbtList#getListType NBT type} of an NBT list is determined
	COMMENT once its first element is inserted; empty NBT lists do not have an associated type.
```

If it does not have an associated type, what would `AbstractNbtList#getListType` return? I think that would also be useful to place in this documentation.

Reply:
It would return `0` iirc, as it was one of the reason for `EndTag` -> `NbtNull`